otel-agent/k8s-helm-windows/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-coralogix-windows
-version: 0.0.1
+version: 0.0.2
```

Comment:
I guess we'll need to add another CI check for the new chart, but I'm happy to do it as a follow up once we also have images published.